SConstruct

```diff
@@ -910,6 +917,17 @@ else:  # GCC, Clang
     if env["werror"]:
         env.Append(CCFLAGS=["-Werror"])
 
+# Configure external includes.
+if env.msvc:
+    if cc_version_major < 16 or (cc_version_major == 16 and cc_version_minor < 10):
+        env.AppendUnique(CCFLAGS=["/experimental:external"])
+    env.AppendUnique(CCFLAGS=["/external:W0"])
+    env["EXTINCPREFIX"] = "/external:I"
+    env["EXTINCSUFFIX"] = ""
+else:
+    env["EXTINCPREFIX"] = "-isystem"
```

Comment:
There's usually a separation between `-isystem` and the path.
```suggestion
    env["EXTINCPREFIX"] = "-isystem "
```

Reply:
Went ahead and put a separator for both prefixes